### DISCUSSION TITLE
Vue/Nuxt examples of auth with JS sdk?

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#6816] [Archive]
Hi there!

I was wondering if there were any nuxt or vue examples available where a simple login flow is demonstrated using the JS Sdk?

Cheers!

**Comment by kees-tce** [comment]
To be more specific, I'm trying to get auth to work using the Directus JavaScript SDK. How can I use the SDK serverside, with users logging in with their own directus credentials. I think I need to override the transport of the SDK to pick up the token from the user client side?

**Reply by rijkvanzanten** [reply]
> How can I use the SDK serverside, with users logging in with their own directus credentials. I think I need to override the transport of the SDK to pick up the token from the user client side?

Your client would send the email and password through a form to your own middleman server that then logs in using the SDK. You would use the json mode for login so you get the access and refresh token on the server layer, cause you can then store the refresh token in a custom session for the user 🤔 

cc @jacobrienstra

**Reply by kees-tce** [reply]
I thought something like that. Just trying to figure out the specifics. Fyi, I'm trying to create a Directus integration for [Vue Storefront](https://docs.vuestorefront.io/v2/)

**Reply by rijkvanzanten** [reply]
Never heard of it, looks interesting! Is your integration publicly accessible somewhere?

**Comment by keesvanbemmel** [comment]
Not yet, I just started and I started with auth :) as soon as that is
working it'll be public

**Reply by kees-tce** [reply]
At the moment I can't even get the json mode to work in a sandbox environment. Even with a prefixed storage it doesn't seem to persist the token
